packages/access-policy/src/AccessPolicy.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+export class AccessPolicy {
```

Comment:
This is an absolutely superficial change. `AccessPolicy` has always been one of the very rare cases where we keep a class in a file named something other than the class name. I find myself constantly forgetting this and struggling to find it when searching in vscode, so I moved it. Hope the reviewer agrees and sorry for doing it here but I didn't feel it warranted a PR of its own 😅